test/packages/missing_required_fields/data_stream/foo/fields/some_fields.yml

```diff
@@ -1,24 +1,18 @@
 - name: source
-  title: Source
-  group: 2
   type: group
   fields:
     - name: geo.city_name
-      level: core
       type: keyword
       description: City name.
       ignore_above: 1024
     - name: geo.location
-      level: core
       type: geo_point
       description: Longitude and latitude.
     - name: geo.region_iso_code
-      level: core
       type: keyword
       description: Region ISO code.
       ignore_above: 1024
     - name: geo.region_name
-      level: core
```

Comment:
@andrewkroh can you also revert the removal of these fields in test packages? So we keep these as they are still used by many integrations. Thanks!